src/library/scala/util/Properties.scala

```diff
@@ -141,16 +141,17 @@ private[scala] trait PropertiesTrait {
   /* Some runtime values. */
   private[scala] lazy val isAvian = javaVmName.contains("Avian")
 
-  private[scala] def coloredOutputEnabled: Boolean = propOrElse("scala.color", "auto") match {
+  private[scala] lazy val coloredOutputEnabled: Boolean = propOrElse("scala.color", "auto") match {
```

Comment:
It's a `def` on 2.13.13, maybe better to keep it that way? Properties can change.